tool/travis.sh

```diff
@@ -130,8 +134,37 @@ install_osx() {
   exit 0
 }
 
-## Automatic Creation of tags and generation of the doxygen documentation #################
 
+run_tests() {
+  if contains "$TARGET" linux; then
+    "${bin}/unit_tests"
+  else
+    echo >&2 "ERROR: UNKNOWN TRAVIS TARGET: ${TARGET}"
+    exit 23
+  fi
+}
+
+# ATTENTION:
+# Please USE the following naming format for any files uploaded to our distribution server
+# <BRANCHNAME>-<BUILDNUMBER>-<TARGET_NAME>.EXTENSION
+# where <PACKAGENAME> is NOT CONTAINING any -
```

Comment:
why do you readd this outdated commit? i guess this is a merge error?

furthermore upload() is not needed anymore, as we don't have a ftp nightly server currently and could readd it if we would

Reply:
@Fohlen @aschaeffer @Fohlen could we please decide if we want to keep docs.inexor.org? It shouldn't be hard to provide ftp for our server if we want to and if we don't want to, then we should remove every link and related code.

This upload function is still there for API docs and only waits for FTP login data to function.